steps/steps-chatwork-message/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: steps-chatwork-message
```

Comment:
Note
I would go with the *Send a Chatwork message* for the title.
What do you think?